docs/concepts/index.md

```diff
@@ -7,27 +7,10 @@ tags:
     - overview
 ---
 
-# Concepts Overview
+# Explore Prefect concepts
 
-Getting started building and running workflows with Prefect doesn't require much more than a knowledge of Python and an intuitive understanding of "tasks" and "flows".  However, deploying and scheduling workflows as well as more advanced usage patterns do require a deeper understanding of the building blocks of Prefect.
+Learn how to develop, deploy, and host Prefect by exploring the concepts in the menu to the left.
 
-These guides are intended to provide the reader with a deeper understanding of how the system works and how it can be used to its full potential; in addition, these guides can be revisited as reference material as you learn more.
+Features specific to [Prefect Cloud](../cloud/) are in their own subheading.
```

Comment:
this seems to be rendering as a heading for some reason?
![Uploading image.png…]()

Reply:
fixed